configure.ac

```diff
@@ -5846,6 +5864,175 @@ else
 fi
 AC_SUBST(TEST_MODULES)
 
+# Modules that are not available on some platforms
+AS_CASE([$ac_sys_system],
+  [AIX], [py_stdlib_not_available="_scproxy spwd"],
+  [VxWorks*], [py_stdlib_not_available="_scproxy _crypt termios grp"],
+  [Darwin], [py_stdlib_not_available=""],
```

Comment:
From the top of my head: `ossaudiodev` and `spwd` is not available on macOS.

Reply:
Thanks! I included that in my latest commit.